app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] Kindly work on this part of the project requirement. The application should ask users for an email confirmation before accessing the app's pages. When I created an account, I was redirected to my page (user's page) which shouldn't be the case.